extra/changelog-generator/test-changelog-generator

```diff
@@ -382,23 +384,27 @@ Should be included N74'
 
 ################################################################################
 
-git commit --allow-empty -m 'Changelog:All: Bump github.com/klauspost/pgzip from 1.2.3 to 1.2.4
+git commit --allow-empty -m 'fix: Bump github.com/klauspost/pgzip from 1.2.3 to 1.2.4
```

Comment:
I don't think we can remove these strings, because we have limited control over what dependabot posts. Have you discussed how to handle this?

Reply:
We do now. Commenting `mark-pr as (type)` on a dependabot PR will amend the commit to this exact format. After enforcing the conventional commit rule, every dependabot PR will most certainly be on this format.

Reply:
Yes, @kacf see https://tracker.mender.io/browse/QA-370

Reply:
👍